docs/api-references/user.md

```diff
@@ -0,0 +1,127 @@
+## User API
+
+### `GET` User
+```
+/api/user/
+```
+Get information about a current user login.
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+#### Sample Response
+```json
+{
+    "data": {
+                "id": 74,
+                "full_name": "dungvan2512",
+                "email": "dungvan@test.email.com",
+                "birthday": null,
+                "gender": 0,
+                "address": null,
+                "phone_number": null,
+                "image": "http://foodmarket.com/images/users/default.jpg",
+                "is_admin": 1,
+                "is_active": 0,
+                "created_at": "2017-08-23 04:13:40",
+                "updated_at": "2017-08-23 04:13:40",
+                "deleted_at": null
+            },
+    "success": true
+}
+```
+
+### `Post` user
+```
+/api/user/create
+```
+Registry a food
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| email | Email | required | email to login |
+| password | Password | required, least at 6 character | password |
+| password_confirmation | Password | required, map to password | password confirmation |
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
```

Comment:
I think this API doesn't need Authorization

Reply:
this API get profile of current user login in client, so we need to Authorization to get data